test/kafka-ingest-open-loop/mzcompose.py

```diff
@@ -119,6 +119,12 @@ def workflow_kafka_ingest_open_loop(
         action="store_true",
         help="whether or not to enable persistence on materialized",
     )
+    parser.add_argument(
+        "--workers",
+        type=int,
+        default=4,
```

Comment:
I think it works if we default to None instead. following down into the Materialized definition, looks like a None omits the --workers flag and we get the default (num cpus), which is my preference for the behavior here

Reply:
Done